controllers/ingressnodefirewallnodestate_controller.go

```diff
@@ -115,3 +122,38 @@ func (r *IngressNodeFirewallNodeStateReconciler) syncIngressNodeFirewallResource
 	}
 	return nil
 }
+
+// addFailSaferules appends failSafe rules to user configured one
+func addFailSaferules(rules *[]infv1alpha1.IngressNodeFirewallProtocolRule) error {
```

Comment:
How does this work with our trie longest prefix first match? Can't a user define some-source/32:tcp/6443 or some-source/32:tcp/22 and that will bypass the failsafe rules?

Reply:
no u can't configure anything against those pinholes because of the webhook

Reply:
ah ok thanks